docs/multi-stage-query/index.md

```diff
@@ -30,7 +30,7 @@ description: Introduces multi-stage query architecture and its task engine
  ingestion method is right for you.
 :::
 
-Apache Druid supports SQL-based ingestion using the bundled [`druid-multi-stage-query` extension](#load-the-extension).
+Apache Druid supports SQL-based ingestion using the bundled `druid-multi-stage-query` extension, which is enabled by default.
```

Comment:
What does enabled by default mean here. 
The extension is part of the example configs. Is that the definition of default here ?

Reply:
Do people generally use the example configs for production deployments?

Maybe this instead:

```suggestion
Apache Druid supports SQL-based ingestion using the bundled `druid-multi-stage-query` extension, which is loaded by default in the example configs if you use `./bin/start-druid` to launch Druid.
```